tests/test_openlibrary.py

```diff
@@ -71,6 +71,18 @@ def test_get_work(self):
         work = self.ol.Work.get(u'OL12938932W')
         self.assertTrue(work.title.lower() == 'all quiet on the western front',
                         "Failed to retrieve work")
+
+    def test_merge_unique_lists(self):
+        test_data = [
+            {'in': [[1,2], [2,3]], 'expect': [1, 2, 3]},
+            {'in': [[1,2,2,2], [2, 2, 3]], 'expect': [1, 2, 3]},
+            {'in': [[9, 10]], 'expect': [9, 10]},
+            {'in': [[1, 1, 1, 1]], 'expect': [1]},
+            {'in': [], 'expect': []},
+            {'in': [[2], [1]], 'expect': [2, 1]}
```

Comment:
Maybe add a case where:
- one of the lists is `[]`
- `hash_fn` is used. Maybe setting the hash_fn to something stupid (`lambda x: 1`), which should always return only first element, and maybe something more normal, too :P